packages/@glimmer/reference/lib/iterable.ts

```diff
@@ -192,17 +189,17 @@ export class IterableReference extends CachedReference<boolean> {
   }
 
   private createIterator(): OpaqueIterator {
-    let { parentRef, key, env } = this;
+    let { parentRef, key } = this;
 
     let iterable = parentRef.value() as { [Symbol.iterator]: any } | null | false;
 
-    let keyFor = makeKeyFor(key, env.getPath);
+    let keyFor = makeKeyFor(key, getPath);
```

Comment:
`makeKeyFor` is not exported, and it is not used again in this file. Any objection to moving to:

```suggestion
    let keyFor = makeKeyFor(key);
```

And simplifying `makeKeyFor` to:

```ts
function makeKeyFor(key: string) {
  // ...snip...
}
```

And `keyForPath` becomes:

```ts
function keyForPath(path: string): KeyFor {
  // ...snip...
}
```